delivery_carrier_label_paazl/models/stock_picking.py

```diff
@@ -73,6 +73,8 @@ def _paazl_generate_labels(self):
         # as we have the labels anyways, attach them
         account = self._get_carrier_account()
         file_type = (account.file_format or "").lower() or "pdf"
+        if self.option_ids:
+            file_type = self.option_ids[0].label_file_type
```

Comment:
Maybe take the file_type from *any* of the options that have a value set, instead of only of the first.

Reply:
Yea I can, anyways was referring to the first record, since vanmoof_delivery allows to set up only one option at a time, but that's vanmoof's logic and we are upstreaming OCA module, then yea it's right thing to do.